### ISSUE TITLE
Build Errors in iOS

### ISSUE THREAD

**Issue by ahmetbicer** [mrousavy/react-native-fast-tflite#14]
### Description

I encountered two errors while attempting to build for iOS with both new and old architectures, each resulting in different issues. When using the new architecture, I encountered the following error:

```shell
node_modules/react-native-fast-tflite/ios/Tflite.h:3:9 'RNTfliteSpec.h' file not found
```

![Screenshot](https://github.com/mrousavy/react-native-fast-tflite/assets/45371442/1926a95c-bc49-4420-a4a3-2b59b9bc404e)

Switching back to the old architecture triggered an another error, which appears to be the same as [this issue](https://github.com/mrousavy/react-native-fast-tflite/issues/10). While a workaround involves adding `arm64` to excluded architectures and using Xcode in Rosetta mode, it's not an ideal solution in my opinion.

```
Building for iOS Simulator, but linking in object file built for iOS, file 'node_modules/react-native-fast-tflite/ios/TensorFlowLiteC.framework/TensorFlowLiteC' for architecture arm64
```

### Steps to reproduce
1. Use the repro provided below.
2. Run a build with `yarn react-native run-ios` (you can toggle the `RCT_NEW_ARCH_ENABLED` flag in Podfile to switch architectures and observe both errors).

### Repro link
[https://github.com/ahmetbicer/fast-tflite-repro](https://github.com/ahmetbicer/fast-tflite-repro)

### React Native version
0.73.1

**Comment by mrousavy** [comment]
Hey - did you fix that error somehow?

**Comment by ahmetbicer** [comment]
Unfortunately, no. I opted for a server-side solution instead. Are you able to successfully run a build with react native 0.73?

**Comment by mrousavy** [comment]
yup, I can build with latest RN.

**Comment by ahmetbicer** [comment]
Ok, It seems like the issue might be on my side. I'll double-check to see if there's anything I missed. Thank you 🙏

**Comment by Qiuzhuang** [comment]
I still run into this problem with 1.1.1. I am in expo 50+xcode 14.3. Error logs are as follows:

› Compiling react-native-fast-tflite Pods/react-native-fast-tflite » Tflite.mm

❌  (/Users/lianqz/workspace/startup/uberapp/node_modules/react-native-fast-tflite/ios/Tflite.h:3:9)

  1 | #ifdef RCT_NEW_ARCH_ENABLED
  2 |
> 3 | #import "RNTfliteSpec.h"
    |         ^ 'RNTfliteSpec.h' file not found
  4 | @interface Tflite : NSObject <NativeTfliteSpec>
  5 | @end
  6 |

› Compiling react-native-fast-tflite Pods/react-native-fast-tflite » TensorflowPlugin.cpp
› Compiling react-native-fast-tflite Pods/react-native-fast-tflite » TensorHelpers.cpp
› Compiling react-native-fast-tflite Pods/react-native-fast-tflite » Promise.cpp
› Compiling react-native Pods/React-RCTImage » RCTImageStoreManager.mm
› Compiling react-native Pods/React-RCTImage » RCTImageLoader.mm
› Compiling react-native Pods/React-RCTFabric » RCTUnimplementedViewComponentView.mm
› Compiling react-native Pods/React-RCTFabric » RCTUnimplementedNativeComponentView.mm
› Compiling react-native Pods/React-RCTFabric » RCTThirdPartyFabricComponentsProvider.mm
› Compiling react-native Pods/React-RCTFabric » RCTTextInputComponentView.mm
› Compiling react-native Pods/React-RCTFabric » RCTSwitchComponentView.mm
› Compiling react-native Pods/React-RCTFabric » RCTScrollViewComponentView.mm
› Compiling react-native Pods/React-RCTFabric » RCTSafeAreaViewComponentView.mm
› Compiling react-native Pods/React-RCTFabric » RCTPullToRefreshViewComponentView.mm
› Compiling react-native Pods/React-RCTFabric » RCTParagraphComponentView.mm
› Compiling react-native Pods/React-RCTFabric » RCTParagraphComponentAccessibilityProvider.mm

› 1 error(s), and 0 warning(s)

CommandError: Failed to build iOS project. "xcodebuild" exited with error code 65.

**Comment by Qiuzhuang** [comment]
BTW, android build pass successfully. :)

**Comment by mrousavy** [comment]
Ah, yea it doesn't work on new arch yet - I probably just need to write the `RNTfliteSpec.h` header but not sure what to put in there yet.

**Comment by giacomocerquone** [comment]
Yes, this issue shouldn't be closed at the moment as a heads-up. I just tried to build an expo project on iOS and it doesn't work. It's working on Android instead

**Comment by lucksp** [comment]
> Yes, this issue shouldn't be closed at the moment as a heads-up. I just tried to build an expo project on iOS and it doesn't work. It's working on Android instead

@Qiuzhuang or @giacomocerquone are you successfully able to run ReactNativeVisionCamera with `useFrameProcessor` with Expo & iOS?

**Comment by mbpictures** [comment]
In case anyone still has this issue (as me), I fixed it by replacing ```"RNTfliteSpec.h"``` by `#import <RNTfliteSpec/RNTfliteSpec.h>`
